### PR TITLE
Update Getting started tutorial to comply documentation style guides

### PR DESCRIPTION
**Pull request by ppbaena**
### Description of the change

Update getting started tutorial to comply with the documentation style guides.

### Benefits

Documentation updated and following style guides.

### Possible drawbacks

N/A

### Applicable issues

- related to #5024 

### Additional information
Update the tutorial info about kubernetes version for Kubeapps (1.21+)

### CODE REVIEW

**Review by castelblanque — APPROVED** [review]
Thanks!

File: site/content/docs/latest/tutorials/getting-started.md [hunk 1]
```diff
@@ -4,13 +4,17 @@ This guide will walk you through the process of deploying Kubeapps for your clus
 
 ## Prerequisites
 
-Kubeapps assumes a working Kubernetes cluster (v1.15+), as well as the [`helm`](https://helm.sh/docs/intro/install/) (3.1.0+) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) command-line interfaces installed and configured to talk to your Kubernetes cluster. Kubeapps has been tested with Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), `minikube` and Docker for Desktop Kubernetes. Kubeapps works on RBAC-enabled clusters and this configuration is encouraged for a more secure install.
+- Kubeapps assumes a working Kubernetes cluster (v1.21+), as well as the [`helm`](https://helm.sh/docs/intro/install/) (3.1.0+) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) command-line interfaces installed and configured to talk to your Kubernetes cluster.
```
Comment (antgamdia) [inline site/content/docs/latest/tutorials/getting-started.md:7]:
```suggestion
- Kubeapps assumes a working Kubernetes cluster (v1.21+), as well as the [`helm`](https://helm.sh/docs/intro/install/) (v3.2.0+) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) command-line interfaces installed and configured to talk to your Kubernetes cluster.
```

**Review by antgamdia — APPROVED** [review]
Thanks

File: site/content/docs/latest/tutorials/getting-started.md (hunk 1 above)
Comment (ppbaena) [reply site/content/docs/latest/tutorials/getting-started.md]: +1 to the suggestion

**Pull request merged**